test/blackbox/run_blackbox_tests/test_optimal_pmtu.h

```diff
@@ -35,17 +35,17 @@ typedef struct pmtu_attr_para {
 } pmtu_attr_para_t;
 
 typedef struct pmtu_attr {
-	pmtu_attr_para_t mtu_sent_probes;
```

Comment:
Why change all the member variables to be pointers to `pmtu_attr_para_t`s? Now you have to allocate, zero and free all of them individually. The previous way seems much more sane to me.
Of course, if you want to avoid copying a `pmtu_attr_t` around, you should pass around pointers to `pmtu_attr_t`s.